pkg/storage/internalstorage/util.go

```diff
@@ -135,9 +135,13 @@ func applyListOptionsToQuery(query *gorm.DB, opts *internal.ListOptions, applyFn
 	// Due to performance reasons, the default order by is not set.
 	// https://github.com/clusterpedia-io/clusterpedia/pull/44
 	for _, orderby := range opts.OrderBy {
-		if supportedOrderByFields.Has(orderby.Field) {
+		orderBy := orderby.Field
```

Comment:
Since there is already a variable `orderby` with a similar name, it might be easier to read by naming it `field` or `column` (`field` maybe better).